isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -401,29 +392,75 @@ namespace Isis {
 
 
   /**
-   * Updates the ControlCubeGraphNode for the measure's serial number to
-   * reflect the addition.  If there is currently no ControlCubeGraphNode for
-   * this measure's serial, then a new ControlCubeGraphNode is created with
+   * Used for verifying graph intergrity
+   *
+   * @returns A string representation of the ControlNet graph
+   */
+  QString ControlNet::GraphToString() const {
+    QString graphString; 
+    typedef boost::graph_traits<Network>::edge_iterator edge_iter;
```

Comment:
I just realized, you should replace the typedef in the header for connectionIterator with this typedef.